Documentation/EyeTracking/EyeTracking_TargetSelection.md

```diff
@@ -2,25 +2,50 @@
 
 # Eye-Supported Target Selection
 This page discusses different options for accessing eye gaze data and eye gaze specific events to select targets in MRTK. 
-Eye Tracking allows for fast and effortless target selections using a combination of information about what a user is looking at with additional inputs such as 
-_hand tracking_ and _voice commands_:
-    - Look & Pinch (i.e., hold up your hand in front of you and pinch your thumb and index finger together)
-    - Look & Say _"Select"_ (default voice command)
-    - Look & Say _"Explode"_ or _"Pop"_ (custom voice commands)
-    - Look & Bluetooth button
-
-## Target Selection
-### Use Generic Focus Handler 
-If Eye Tracking is set up correctly (see [Basic MRTK Setup to use Eye Tracking](EyeTracking_BasicSetup.md)), enabling users to select 
+Eye tracking allows for fast and effortless target selections using a combination of information about what a user is looking at with additional inputs such as _hand tracking_ and _voice commands_:
+- Look & Pinch (i.e., hold up your hand in front of you and pinch your thumb and index finger together)
+- Look & Say _"Select"_ (default voice command)
+- Look & Say _"Explode"_ or _"Pop"_ (custom voice commands)
+- Look & Bluetooth button
+
+To select holographic content using eye gaze, there are several options:
+
+**1. Use the primary focus pointer:** 
+This can be understood as your primary cursor. 
+Please note that based on the current design head or eye gaze is suppressed as a cursor input if hand rays are used.
+
+For example: 
+A user wants to select a distant holographic button.
+As a developer, you want to provide a flexible solution that allows the user to achieve this tasks in various conditions:
+- Walk up to the button and poke it
+- Look at it from a distance and say "select"
+- Target the button using a hand ray and performing a pinch
+In this case, the most flexible solution is to use the primary focus handler as it will notify you whenever the currently prioritized primary focus pointer triggers an event. 
+Please note that if hand rays are enabled, the head or eye gaze focus pointer are disabled as soon as the hands come into view.
+
+This is why in our eye tracking sample scenes, we have disabled the hand ray to allow for showcasing richer interactions using eyes + hand motions - see [**Eye-Supported Positioning**](EyeTracking_Positioning.md).
+
+**2. Use both eye focus and hand rays at the same time:** 
+There might be instances where you want to be more specific which type of focus pointers can trigger certain events and allow for simultaneously using multiple far interaction techniques. 
+
+For example:
+In your app, a user can use far hand rays to manipulate some holographic mechanical setup.
```

Comment:
Reading this section, I'm a little confused above this example, because it sounds like you're not really using both far hand rays and eye focus at the same time - it looks like you're using far hand rays to do some setup, and then you're switching to near interaction and eyes.

The confusing part about this section is that other parts of the docs allude to how having far hands up will turn off eye stuff, but this section is named such that it seems like it would be supported.

Reply:
I guess with a title "Use both eye focus and hand rays at the same time" I would imagine they are both using eyes to look at a checklist AND doing far hand ray manipulation at the same time, but that isn't supported based on current interaction rules.

Reply:
It is supported using the EyeTrackingTarget now as it's not based on the FocusHandler anymore. This means that users could grab two holograms with their hand rays and still scroll through text using their eyes, because the auto scroll script is set up to support this unique instance. However, this should not be a default and this is why we didn't simply enable eyes to be supported when hand rays are active, as this could also be confusing if using voice commands and where to channel them to. 
Long story short: Yep, I should explain it better :)